src/oemof/solph/_models.py

```diff
@@ -129,6 +139,18 @@ def __init__(self, energysystem, **kwargs):
         ]
 
         self.flows = self.es.flows()
+        if self.iscellular:
+            for cell in self.ec:
+                for io, f in cell.flows().items():
+                    if io not in self.flows.keys():
+                        self.flows.update({io: f})
+                    else:
+                        msg = (
+                            "Two flows with identical input-output are tried "
+                            "to be added to the Model. Second flow is skipped."
+                            "\n{}"
+                        )
+                        raise Warning(msg.format({io: f}))
```

Comment:
I'm not so sure if raising an error is enough or whether it should be an error.

Reply:
Good point. I'm changing it to be a ValueError.